src/plugins/intel_gpu/src/graph/impls/onednn/convolution_onednn.cpp

```diff
@@ -318,25 +318,17 @@ struct convolution_onednn : typed_primitive_onednn_impl<convolution> {
             _attrs->set_zero_points(DNNL_ARG_WEIGHTS, 0, dnnl::memory::dims{}, _wzp_data_type);
         }
 
-        if (zero_bias) {
-            auto prim_desc = std::make_shared<dnnl::convolution_forward::primitive_desc>(
-                                    ib.get_engine().get_onednn_engine(),
-                                    dnnl::prop_kind::forward_inference, dnnl::algorithm::convolution_direct,
-                                    input_md, weights_md, output_md,
-                                    strides, dilates, padding_l, padding_r,
-                                    *_attrs.get());
-            _pd = *prim_desc;
-        } else {
-            auto bias_md = onednn::layout_to_memory_desc(impl_params->get_input_layout(2), dnnl::memory::format_tag::any, true);
-            auto prim_desc = std::make_shared<dnnl::convolution_forward::primitive_desc>(
-                                    ib.get_engine().get_onednn_engine(),
-                                    dnnl::prop_kind::forward_inference, dnnl::algorithm::convolution_direct,
-                                    input_md, weights_md, bias_md, output_md,
-                                    strides, dilates, padding_l, padding_r,
-                                    *_attrs.get());
-            _pd = *prim_desc;
+        dnnl::memory::desc bias_md = nullptr;
```

Comment:
nit: `dnnl::memory::desc bias_md;` will be more explicit for `zero_md` according to onednn api.

Reply:
My point is to replace 
`dnnl::memory::desc bias_md = nullptr;` to `dnnl::memory::desc bias_md;` for explicit use of API. They seems to be basically same.

Reply:
Actually, I understood and tried as you suggested. I couldn't prove perfectly why, but the cache works properly only when `nullptr `is explicitly assigned. I didn't compare memory stack both when `nullptr `is explicitly assigned and when it isn't, using any specific tool. However, I confirmed through many tests that cache works properly only when `nullptr `is explicitly assigned.